### DISCUSSION TITLE
Remap Mouse Click to Keyboard Key

### DISCUSSION THREAD

**Discussion by ranvierion** [pqrs-org/Karabiner-Elements#3928] [Q&A]
I attempted this solution on: https://karabiner-elements.pqrs.org/docs/help/how-to/mouse-button/

I wanted to map a left mouse button press to a keyboard button press but to no avail.

I connected a USB mouse to the MacBook Pro and the mouse was detected in the, "Device" panel. 

I enabled "Modify events" button for this mouse as per the instructions in the link above.

I visited, "Simple modification" and choose this mouse.

I selected, "Letter Keys" > "a" for the left dropdown.

I selected, "mouse button" > "button1"

I have mapped, "a" to trigger an action inside a software that listens to the keyboard.
I also went into Microsoft Word and use the left mouse button but I don't see, "a" being type out as wel.

But no matter how I press on my left mouse button, "a" was not triggered.

**Comment by ranvierion** [comment]
**UPDATE:**

I changed the LEFT dropdown to "button2" and the RIGHT dropdown option to "a"
And I have successfully map, "a" to the RIGHT MOUSE button.

But my question remains, I CANNOT FIND, "button1" at MOUSE BUTTONS options > 

Because I want left mouse click to trigger, "a"

**Comment by madewokherd** [comment]
On the left sidebar, click on "Expert" and then "Enable unsafe configuration". Just make sure you leave yourself a way to click on things.
